platform/view/sdk/sdk.go

```diff
@@ -216,7 +212,7 @@ func (p *p) registerViewServiceServer() error {
 func (p *p) initGRPCServer() error {
 	configProvider := view.GetConfigService(p.registry)
 
-	listenAddr := configProvider.GetString("fsc.listenAddress")
+	listenAddr := configProvider.GetString("fsc.address")
```

Comment:
Probably something that needs testing for the whole core file but what happens if fsc.address is not specified ?